src/sentry/utils/services.py

```diff
@@ -135,8 +157,6 @@ def copy(self):
     None,
 ]
 
-T = TypeVar("T")
```

Comment:
It seems like we always pass in `Service` backends here so seemed to make more sense to be explicit.